shopinvader/services/invoice.py

```diff
@@ -85,3 +85,10 @@ def _get_report_action(self, target, params=None):
         :return: dict/action
         """
         return target.invoice_print()
+
+
+class DeprecatedInvoiceService(Component):
+    _inherit = "shopinvader.invoice.service"
+    _name = "shopinvader.deprecated.invoice.service"
+    _usage = "invoice"
+    _description = "Deprecated Service use 'invoices' instead"
```

Comment:
Shouldn't we log something? Maybe at module load to not pollute std output every time the components are used

Reply:
it will be great but I do not know how to do this.
If we have an idea we can implement it latter

Reply:
Eg: override dispatch?

Reply:
or better at `__init__`?

Reply:
If we override the dispatch, the warning will be displayed each time the deprecated service is called.... IMO it's not a problem but this could generate a lot of burden on existing websites.